### PR TITLE
feat: add faq for accelerate transaction billing #DA-935

### PR DESCRIPTION
**Pull request by ankur-arch**

### CODE REVIEW

File: content/300-accelerate/600-faq.mdx [hunk 1]
```diff
@@ -125,3 +125,27 @@ In the rare event of a service disruption, falling back to a direct connection w
 If there is a service disruption, it's recommended to verify on the [status page](https://pris.ly/data-platform-status). You can reach out to one of Prisma's [support channels](/platform/support) for assistance.
 
 > **Note:** Additionally, it's worth noting that some edge function runtime environments may not support direct connections with Prisma ORM. For further details, refer to our [Edge functions documentation](/orm/prisma-client/deployment/edge/overview).
+
+Sure! Here's the revised FAQ item:
```
Comment (mhessdev) [inline content/300-accelerate/600-faq.mdx]: Why is this here? Forget to remove the chatgpt part?

**Review by mhessdev — CHANGES_REQUESTED** [review]

File: content/300-accelerate/600-faq.mdx [hunk 2]
```diff
@@ -125,3 +125,27 @@ In the rare event of a service disruption, falling back to a direct connection w
...
+However, when using the [`$transaction` API for sequential client operations](/orm/prisma-client/queries/transactions#sequential-prisma-client-operations), regardless of the number of queries within the array, it counts as only one billable query. For example:
+
+```ts
+await prisma.$transaction([
+  prisma.user.deleteMany({ where: { name: 'John Doe' } }),
+  prisma.user.createMany({ data }),
+]);
+```
+
+If you don't need the additional functionality of [interactive transactions](/orm/prisma-client/queries/transactions#interactive-transactions), you can save costs by relying on [array-based transactions](/orm/prisma-client/queries/transactions#sequential-prisma-client-operations) instead.
```
Comment (rtbenfield) [inline content/300-accelerate/600-faq.mdx]: The array format also performs better on Accelerate as it can execute as one round-trip to the database, whereas interactive transactions necessitate a round-trip to for start, commit, and each individual operation on the transaction. This may be worth calling out as another incentive.

File: content/300-accelerate/600-faq.mdx (hunk 1 above)
Comment (ankur-arch) [reply content/300-accelerate/600-faq.mdx]: My bad.

Comment (ankur-arch) [inline content/300-accelerate/600-faq.mdx]:
```suggestion
```

File: content/300-accelerate/600-faq.mdx (hunk 2 above)
Comment (ankur-arch) [reply content/300-accelerate/600-faq.mdx]: Thanks @rtbenfield , will add that in too.

**Review by mhessdev — DISMISSED** [review]

File: content/300-accelerate/600-faq.mdx [hunk 3]
```diff
@@ -125,3 +125,25 @@ In the rare event of a service disruption, falling back to a direct connection w
...
+However, when using the [`$transaction` API for sequential client operations](/orm/prisma-client/queries/transactions#sequential-prisma-client-operations), regardless of the number of queries within the array, it counts as only one billable query. For example:
+
+```ts
+await prisma.$transaction([
+  prisma.user.deleteMany({ where: { name: 'John Doe' } }),
+  prisma.user.createMany({ data }),
+]);
+```
+
+If you don't need [interactive transactions](/orm/prisma-client/queries/transactions#interactive-transactions), you can save costs and improve performance by using [array-based transactions](/orm/prisma-client/queries/transactions#sequential-prisma-client-operations). Array-based transactions perform better on Accelerate because they execute in one round-trip to the database, while interactive transactions require separate round-trips for start, commit, and each individual operation on the transaction.
```
Comment (ankur-arch) [inline content/300-accelerate/600-faq.mdx]: @rtbenfield added the perf incentive here aswell. Let me know if it looks good to you.

Comment (rtbenfield) [reply content/300-accelerate/600-faq.mdx]: This looks great 👍🏻  I think the "sequential operations transaction API" name you had previously is what the ORM docs use as well as ["$transaction([]) API"](https://www.prisma.io/docs/orm/prisma-client/queries/transactions#transaction-api). I just call it the array format because that is usually more recognizable what I'm referring to. I'm not sure which is more correct in this case.

Comment (ankur-arch) [reply content/300-accelerate/600-faq.mdx]:
```suggestion
If you don't need [interactive transactions](/orm/prisma-client/queries/transactions#interactive-transactions), you can save costs and improve performance by using [sequential operations transactions](/orm/prisma-client/queries/transactions#sequential-prisma-client-operations). Sequential operations transactions perform better on Accelerate because they execute in one round-trip to the database, while interactive transactions require separate round-trips for start, commit, and each individual operation on the transaction.
```

**Review by rtbenfield — APPROVED** [review]
